### PR TITLE
WIP Hub widget: initial commit

### PR DESCRIPTION
**Pull request by janezd**
As advertised in https://docs.google.com/document/d/1ulaWDXqB6RFPOOFvaSc8cnOs0yLwVYfMYrKov4ueUi0/edit?usp=sharing

### CODE REVIEW

File: orangecontrib/prototypes/widgets/owhub.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    description = "Universal hub widget"
+    icon = "icons/Hub.svg"
+
+    inputs = [("Object", object, "get_input", widget.Default)]
+    outputs = [("Object", object, widget.Dynamic)]
+
+    NOTHING = "Nothing on input"
+
+    def __init__(self, parent=None):
+        super().__init__(parent)
```
Comment (astaric) [inline orangecontrib/prototypes/widgets/owhub.py]: OWWidget's init does not have parameters in Orange3.
https://github.com/biolab/orange3/blob/master/Orange/widgets/widget.py#L142

Setting parent (by canvas) is handled in widget's __new__ method.

**Comment by kernc** [comment]
That spec text is excerpt from a future advertorial blog post, I hope.

File: orangecontrib/prototypes/widgets/owhub.py (hunk 1 above)
Comment (janezd) [inline orangecontrib/prototypes/widgets/owhub.py]: Fixed. I actually copied this from somewhere; it turns out many widgets still have arguments - some even signalManager - in __init__

**Comment by janezd** [comment]
The text is for blog, if we decide so. The whole widget is a bit of a joke, but it is also quite practical (i.e. practical joke :). In lectures I would need it exactly for the purpose described in the "advertisement" (top part, the bottom is not serious).

**Pull request merged**